src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests.cs

```diff
@@ -89,7 +90,7 @@ public unsafe partial class A : C, I
             var source = CreateCompilationWithMscorlibAndSystemCore(code);
 
             // the following should not crash
-            source.GetDiagnosticsForSyntaxTree(CompilationStage.Compile, source.SyntaxTrees[0], null, true);
+            source.GetDiagnosticsForSyntaxTree(CompilationStage.Compile, source.SyntaxTrees[0], null, true, includeDiagnosticsWithSourceSuppression: false, cancellationToken: CancellationToken.None);
```

Comment:
This changes will not be needed as I plan to add a compilation option for this flag. I'll revert this when I send that PR out.